airbyte-webapp/src/config/frequencyConfig.ts

```diff
@@ -1,6 +1,6 @@
-import { ConnectionSchedule } from "core/request/AirbyteClient";
+import { ConnectionScheduleDataBasicSchedule } from "core/request/AirbyteClient";
 
-export const frequencyConfig: Array<ConnectionSchedule | null> = [
+export const frequencyConfig: (ConnectionScheduleDataBasicSchedule | null)[] = [
```

Comment:
Not sure why the linter didn't catch this but this should be the way it was before:
```
src/config/frequencyConfig.ts
  Line 3:31:  Array type using 'T[]' is forbidden for non-simple types. Use 'Array<T>' instead  @typescript-eslint/array-type
```

Reply:
This was at my recommendation. I didn't realize  we were using `Array<>` for more complex types.